sdk/core/azure-core/inc/azure.hpp

```diff
@@ -5,10 +5,8 @@
 
 #include <internal/contract.hpp>
 
-#define AZURE_UNREFERENCED_PARAMETER(x) ((void) (x));
+#define AZURE_UNREFERENCED_PARAMETER(x) ((void)(x));
```

Comment:
I don't think we need this. Why not simply `(void)parameter`? It is a language-native way to indicate that a thing is unused, and it does not vary between platforms.

Reply:
shouldn't have change this file at all. Looks like my autoFormatter found it and changed it. I will remove